.scalafmt.conf

```
@@ -1,6 +1,7 @@
 version = 3.8.3
 
-runner.dialect = scala213
+runner.dialect = scala3
```

as long as we still have Scala 2.13 sources, wouldn't it be meaningful to have scalafmt support for both 2.13 and 3?

---

What we've done in search-services is to switch to the subset compatible by both compilers: https://github.com/commercetools/search-services/pull/1432

```suggestion
runner.dialect = Scala213Source3
```

---

The Scala213Source3 doesn't seem to tolerate the Scala 3 syntax 🤔 
It failed on `given`, `derives` and `inline`. So Switching back to Scala3. It seems to deal quite OK with scala 2 anyway.

---

Ah, my bad. We didn't have Scala 3 only syntax yet which is why we didn't run into it.